kloppy/infra/serializers/tracking/epts/metadata.py

```diff
@@ -40,7 +40,7 @@ def _load_provider_parameters(parent_elm, value_mapper=None) -> Dict:
 
 def _load_periods(global_config_elm, frame_rate: int) -> List[Period]:
     provider_params = _load_provider_parameters(
-        global_config_elm.find("ProviderGlobalParameters"), value_mapper=int
```

Comment:
This was not needed as the result is the same whether you map to int or not, and it made the serialization crashed when there where provider parameters present that where not numbers.

Reply:
Make sense!